docs/docsource/reference/apis/cloudserver/bucket_operations/put_bucket_notification_configuration.rst.txt

```diff
@@ -24,7 +24,7 @@ By default, your bucket has no event notifications configured.
 This operation replaces the existing notification configuration with the
 configuration you include in the request body.
 
-When Zenko receives this request, it verifies that the aws sqs Kafka destination
+When XDM  receives this request, it verifies that the aws sqs Kafka destination
```

Comment:
Should the `XDM` references in this file also be `|product|` ?

Reply:
This was my hope, yes...